--- ./src/acl/ConnMark.cc	2021-05-09 16:34:24.031458905 +0200
+++ ./src/acl/ConnMark.cc	2021-05-09 16:34:35.928598480 +0200
@@ -15,6 +15,7 @@
 #include "Debug.h"
 #include "http/Stream.h"
 #include "sbuf/Stream.h"
+#include <limits>
 
 bool
 Acl::ConnMark::empty() const

--- ./src/security/ServerOptions.cc	2021-05-09 16:36:41.020234991 +0200
+++ ./src/security/ServerOptions.cc	2021-05-09 16:37:02.434418712 +0200
@@ -23,6 +23,7 @@
 #include <openssl/err.h>
 #endif
 #endif
+#include <limits>
 
 Security::ServerOptions &
 Security::ServerOptions::operator =(const Security::ServerOptions &old) {
