diff --git a/src/plugins/media-export/rygel-media-export-dvd-container.vala b/src/plugins/media-export/rygel-media-export-dvd-container.vala
index d1cb38b7137c1f1222a9e5254b05b5058a87d93d..a692c30c75e619aa14bcd21e9f28b61f2deed784 100644
--- a/src/plugins/media-export/rygel-media-export-dvd-container.vala
+++ b/src/plugins/media-export/rygel-media-export-dvd-container.vala
@@ -54,7 +54,7 @@ internal class Rygel.MediaExport.DVDContainer : SimpleContainer, UpdatableObject
                                         Xml.ParserOption.NOWARNING |
                                         Xml.ParserOption.NOBLANKS |
                                         Xml.ParserOption.NONET);
-        this.doc = new GUPnP.XMLDoc (doc);
+        this.doc = new GUPnP.XMLDoc ((owned) doc);
 
         var context = new Xml.XPath.Context (this.doc.get_doc ());
         var xpo = context.eval ("/lsdvd/track");
